meinberlin/apps/dashboard2/__init__.py

```diff
@@ -0,0 +1,3 @@
+from .contents import content
+
+__all__ = ['content']
```

Comment:
this is to be consistent with phases.content and allows us to 
`from adhocracy4.phases import content` instead of `from adhocracy4.phases.content import content`
resp 
`from meinberlin.dashboard2 import content` instead of `from meinberlin.dashboard2.content import content`